tcclient/http.go

```diff
@@ -47,6 +48,18 @@ type CallSummary struct {
 	Attempts int
 }
 
+// utility function to create a URL object based on given data
+func setURL(connectionData *ConnectionData, route string, query url.Values) (u *url.URL, err error) {
+	u, err = url.Parse(connectionData.BaseURL + route)
+	if err != nil {
+		return nil, fmt.Errorf("Cannot parse url: '%v', is BaseURL (%v) set correctly?\n%v\n", connectionData.BaseURL+route, connectionData.BaseURL, err)
+	}
+	if query != nil {
+		u.RawQuery = query.Encode()
+	}
+	return
```

Comment:
It looks like this method is supposed to return a url.URL and error....but this return statement doesn't have anything.  Are there implicit returns in Go?